### ISSUE TITLE
f5_openstack_agent.lbaasv2.drivers.bigip.system_helper.purge_folder_contents does not properly delete LTM virtual addresses

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-openstack-agent#669]
#### OpenStack Release
9.2.0

#### Description
Calling purge_folder_content properly removes everything but virtual addresses. If you try and remove the folder using delete_folder it does not work because the folder is not empty. 

#### Agent Version
9.2.0+

#### Operating System
CentOS 7.3

#### OpenStack Release
NA

#### Deployment
Create a tenant loadbalancer, stop the agent, try to purge_folder_contents. 

If you call purge_folder_contents, then run:

```
vaddrs = bigip.tm.ltm.virtual_address_s.get_collection()
for va in vaddrs:
    if va.partition == folder.name:
        va.delete()
```
 and then delete_folder, the folder deletes properly.

**Comment by jgruber** [comment]
Low priority since we have a work-around to build into tools.

**Comment by richbrowne** [comment]
Janaki, could you verify that this issue is resolved, by writing a test for it?

**Comment by richbrowne** [comment]
Fixed by bb9983d54b3a0727da56822cdc988761511ef696
